dsperse/src/backends/onnx_models.py

```
@@ -20,45 +20,91 @@ def run_inference(input_file: str, model_path: str, output_file: str):
+            raw_output = session.run(None, input_dict)
+
+            model_outputs = session.get_outputs()
+            output_tensors = {}
+            for i, out in enumerate(model_outputs):
+                output_tensors[out.name] = torch.tensor(raw_output[i], dtype=torch.float32)
+
+            output_tensor = torch.tensor(raw_output[0], dtype=torch.float32)
+            result = RunnerUtils.process_final_output(output_tensor)
+            result['output_tensors'] = output_tensors
+            RunnerUtils.save_to_file_flattened(result['logits'], output_file)
+            return True, result
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)